Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -1468,8 +1468,13 @@
 #endif
 #endif
 
+#if ENABLE(BLOCK_SET_EXCEPTION_PORTS)
+        (with-filter (require-not (webcontent-process-launched))
```

Comment:
Should we not have the `lockdown-mode` case for launch-time LDM? Or is LDM only true after we've launched?

Reply:
Good point, I think we always want to block it in LDM, which is the current behavior.

Thanks for reviewing!